tests/integration/test_fim/test_files/test_audit_buffer/test_audit_buffer_over_time.py

```diff
@@ -0,0 +1,323 @@
+                                restart_syscheck_function, wait_syscheck_start):
+    '''
+    description: This test validates the behavior of "queue_size" in tandem with "max_eps". Check that when files are
+                 added causing whodata queue to overflow, and after some files are processed, if new files are added
+                 that do not exceed the empty space in the queue, only the files from the first insertion, that caused
+                 the overflow are detected in scheduled mode. All files from second insertion are detected in whodata.
+
+    test_phases:
+        - setup:
+            - Set wazuh configuration and local_internal_options.
+            - Create custom folder for monitoring
+            - Clean logs files and restart wazuh to apply the configuration.
+        - test:
+            - Insert enough files to fill queue
+            - Detect if whodata queue has overflowed
+            - Wait x seconds for space to be freed in queue
+            - Insert files a second time
+            - Validate only files from the first insert were detected in scheduled mode
+            - Validate a all files from the second insert are detected.
+        - teardown:
+            - Delete custom monitored folder
+            - Restore configuration
+            - Stop wazuh
+
+    wazuh_min_version: 4.5.0
+
+    tier: 2
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration values for ossec.conf.
+        - metadata:
+            type: dict
+            brief: Test case data.
+        - test_folders:
+            type: dict
+            brief: List of folders to be created for monitoring.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set ossec.conf configuration.
+        - create_monitored_folders_module:
+            type: fixture
+            brief: Create a given list of folders when the module starts. Delete the folders at the end of the module.
+        - configure_local_internal_options_function:
+            type: fixture
+            brief: Set local_internal_options.conf file.
+        - restart_syscheck_function:
+            type: fixture
+            brief: restart syscheckd daemon, and truncate the ossec.log.
+        - wait_syscheck_start:
+            type: fixture
+            brief: check that the starting FIM scan is detected.
+
+    assertions:
+        - Verify when queue is full an event informs audit events may be lost
+        - Verify when queue is full at start up audit healthcheck fails and does not start
+        - Verify when using invalid values an error message is shown and does not start
+        - Verify configured queue_size value
+        - Verify real-time whodata thread is started correctly
+
+    input_description: The file 'configuration_audit_buffer_values' provides the configuration template.
+                       The file 'cases_audit_buffer_values.yaml' provides the tes cases configuration
```

Comment:
Typo in description.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4227/commits/818122a07a426d23dde0fb26954da571040cd55b

Reply:
I don't see this change applied.